### PR TITLE
new-log-viewer: Override JoyUI `fontFamily` and `radius` properties globally.

### PR DESCRIPTION
**Pull request by junhaoliao**
# References

new-log-viewer series: #45 #46 #48 #51 #52 #53 #54 #55 #56 #59 #60 #61 #62 #63 #66 #67 #68 #69 #70 #71 #72 #73 #74 #76 #77 #78 #79 #80 #81 #82

# Description

1. Centralize border radius overrides with "radius" property in JoyUI theme config.

# Validation performed

1. Launched debug server and verified component border radius matching expected overridden value. Used the browser debugger to "Inspect" individual elements to confirm.

## Summary by CodeRabbit

- **New Features**
  - Introduced a new border radius property for UI components, enhancing the visual consistency across various sizes.

- **Improvements**
  - Updated focus property for better accessibility, setting a more appropriate thickness.
  - Enhanced theme configuration for improved light and dark mode support, ensuring a cohesive user experience.

- **Bug Fixes**
  - Removed unnecessary style overrides from specific components, streamlining their appearance.

### CODE REVIEW

**Comment by Henry8192** [comment]
lgtm. A minor issue is `outLineWidth` seems to change nothing in our UI, even before the pr change.

**Review by Henry8192 — APPROVED** [review]

**Comment by junhaoliao** [comment]
> A minor issue is outLineWidth seems to change nothing in our UI, even before the pr change.

You're right. Could you check if e5186ab43bb7306d2939c509bf88ce4773003e5b correctly does the override?

File: new-log-viewer/src/components/theme.tsx [hunk 1]
```diff
@@ -83,6 +56,21 @@
             },
         },
     },
+    focus: {
+        thickness: "1px",
+    },
```
[the remark this answers is not in the document]
Comment (junhaoliao) [reply new-log-viewer/src/components/theme.tsx:61]: Considering the color contrast, I feel a `1px` focus ring can be deemed sufficient.

**Comment by Henry8192** [comment]
For the title, how about: new-log-viewer: Apply `ylv-font-family` and override `radius` property globally

**Comment by junhaoliao** [comment]
> For the title, how about: new-log-viewer: Apply `ylv-font-family` and override `radius` property globally

How about
```
new-log-viewer: Override JoyUI `fontFamily` and `radius` properties globally.
```

**Review by Henry8192 — APPROVED** [review]
LGTM.

**Pull request merged**